config/crd/bases/v1/datadoghq.com_datadogagentprofiles.yaml

```
@@ -0,0 +1,109 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.1
+  creationTimestamp: null
+  name: datadogagentprofiles.datadoghq.com
+spec:
+  group: datadoghq.com
+  names:
+    kind: DatadogAgentProfile
+    listKind: DatadogAgentProfileList
+    plural: datadogagentprofiles
+    shortNames:
+      - dap
+    singular: datadogagentprofile
+  scope: Namespaced
+  versions:
+    - name: v1alpha1
+      schema:
+        openAPIV3Schema:
+          description: DatadogAgentProfile is the Schema for the datadogagentprofiles API
+          properties:
+            apiVersion:
+              description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+              type: string
+            kind:
+              description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: DatadogAgentProfileSpec defines the desired state of DatadogAgentProfile
+              properties:
+                config:
+                  properties:
+                    override:
```

This looks good 👍🏼 
And if we do want to include features in the CRD, I suppose it'll look like this? 

```suggestion
                config:
                  properties:
                    features:
                      [...]
                    override:
                      [...]           
```

---

I think so. To follow the same structure as the DDA config.

---

It would be:
```yaml
spec:
  config:
    override:
```
`spec` would be the DAP spec and `config.override` would be like `spec.override` from the DDA

---

ok so `spec.override` would include override of DDA features, got it 👍🏼

---

Features would be under `config`, but not under `override`, to match the `spec` section of dda v2alpha1:
```yaml
spec:
  config:
    override:
      [...]
    features:
      [...]
    global:
      [...]
```